### PR TITLE
Extending fraud detection project with online feature validation tutorial

### PR DESCRIPTION
**Pull request by pyalex**
Signed-off-by: Oleksii Moskalenko <moskalenko.alexey@gmail.com>

### CODE REVIEW

File: notebooks/Validating_Online_Features_While_Detecting_Fraud.ipynb [hunk 1]
```diff
@@ -0,0 +1,1296 @@
...
+   },
+   "outputs": [],
+   "source": [
+    "%env COMPILE_GO=True\n",
+    "%env FEAST_USAGE=False\n",
+    "\n",
+    "!pip install 'feast[gcp]'\n",
+    "!pip install 'feast[redis]'\n",
+    "!pip install 'feast[ge]'\n",
+    "!pip install 'feast[go]'\n",
```
Comment (achals) [inline notebooks/Validating_Online_Features_While_Detecting_Fraud.ipynb]: Why not just collapse into a single install:
```
!pip install 'feast[gcp,ge,go,redis]'
```

File: notebooks/Validating_Online_Features_While_Detecting_Fraud.ipynb [hunk 2]
```diff
@@ -0,0 +1,1296 @@
...
+   "metadata": {},
+   "source": [
+    "### Making validation fail"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, if some invalid data, that doesn't met our expectations, will be ingested into the online store and then retrieved via the feature server we should ovbserve how validation is failing."
```
Comment (achals) [inline notebooks/Validating_Online_Features_While_Detecting_Fraud.ipynb]: nit, `observe`

File: notebooks/Validating_Online_Features_While_Detecting_Fraud.ipynb [hunk 3]
```diff
@@ -0,0 +1,1296 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "end_ts = datetime.now()\n",
+    "start_ts = end_ts - timedelta(minutes=10)\n",
+    "\n",
+    "! feast validate --feature-service user_features \\\n",
```
Comment (achals) [inline notebooks/Validating_Online_Features_While_Detecting_Fraud.ipynb:1115]: Will the space between `!` and `feast` cause any problems?

**Review by achals — APPROVED** [review]

File: notebooks/Validating_Online_Features_While_Detecting_Fraud.ipynb (hunk 3 above)
Comment (pyalex) [reply notebooks/Validating_Online_Features_While_Detecting_Fraud.ipynb:1115]: inherited this style from the original fraud tutorial. This works and seems a little cleaner to me.

File: notebooks/Validating_Online_Features_While_Detecting_Fraud.ipynb (hunk 2 above)
Comment (pyalex) [reply notebooks/Validating_Online_Features_While_Detecting_Fraud.ipynb]: fixed

File: notebooks/Validating_Online_Features_While_Detecting_Fraud.ipynb (hunk 1 above)
Comment (pyalex) [reply notebooks/Validating_Online_Features_While_Detecting_Fraud.ipynb]: replaced

**Pull request merged**